DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/use_rtl_dsp/README.md

```diff
@@ -0,0 +1,242 @@
+# Using RTL with customised DSP Block
+
+This FPGA tutorial demonstrates how to build SYCL device libraries from RTL sources and use them in your SYCL design.
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | 
+| Hardware                          | Intel® Agilex® 7, Arria® 10, and Stratix® 10 FPGAs
+| Software                          | Intel® oneAPI DPC++/C++ Compiler
+| What you will learn               | How to integrate customised DSP block RTL into your oneAPI program and emulate it using a C model, as well as pulling the RTL directly into your full system design.
+| Time to complete                  | 30 minutes
+
+> **Note**: Even though the Intel DPC++/C++ OneAPI compiler is enough to compile for emulation, generating reports and generating RTL, there are extra software requirements for the simulation flow and FPGA compiles.
+>
+> For using the simulator flow, Intel® Quartus® Prime Pro Edition and one of the following simulators must be installed and accessible through your PATH:
+> - Questa*-Intel® FPGA Edition
+> - Questa*-Intel® FPGA Starter Edition
+> - ModelSim® SE
+>
+> When using the hardware compile flow, Intel® Quartus® Prime Pro Edition must be installed and accessible through your PATH.
+>
+> :warning: Make sure you add the device files associated with the FPGA that you are targeting to your Intel® Quartus® Prime installation.
+
+
+> :warning: When targeting an IP-only flow, the RTL library feature will not work when compiling to Quartus and will error out in the late-stage compile. RTL libraries will work in the simulation flow. This will be fixed in a future release. This is documented in the [compiler release notes](https://www.intel.com/content/www/us/en/developer/articles/release-notes/intel-oneapi-dpc-c-compiler-release-notes.html).
+
+## Prerequisites
+
```

Comment:
```suggestion
```

Reply:
is there any particular reason of removing the Prerequisites title? I see that other code sample still has this section remained.